code/modules/uplink/uplink_items.dm

```diff
@@ -1110,7 +1110,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			load on the grid, causing a station-wide blackout. The sink is large and cannot be stored in most \
 			traditional bags and boxes."
 	item = /obj/item/powersink
-	cost = 6
+	cost = 8
```

Comment:
???????? This is pretty loud and has pretty easy counterplay

Reply:
Fair, just seems robust